Dockerfile

```diff
@@ -1,14 +1,17 @@
-ARG ALPINE=alpine:latest
-FROM ${ALPINE} AS alpine
+FROM registry.suse.com/bci/bci-base:15.3.17.17.8 AS build
+
 ARG ARCH
 ARG KUBERNETES_RELEASE=v1.21.3
 WORKDIR /bin
 RUN set -x \
- && apk --no-cache add curl \
+ && zypper -n install curl \
  && curl -fsSLO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/${ARCH}/kubectl \
  && chmod +x kubectl
+RUN echo "kubectl:x:1000:1000::/home/kubectl:/bin/bash" > /etc/passwd
```

Comment:
Ok - so in that post that you linked they used the useradd command and then copied over the resulting file in it's entirety. Rather than what we did here - which did not do that. 

There's two things that I am concerned about here:
1) Removing the rest of the users from /etc/passwd will lead to some level of instability
2) Useradd is also doing something else behind the scenes (like creating a home folder) that we aren't doing/will need to do manually.

In other more restricted environments I'm ok with us doing this, but if we have the well-tested binaries available I would prefer to use them rather than manually making the edits ourselves. 

Let me know what you think.

Reply:
I think as long as we copy over /etc/passwd and /etc/group we should be fine.